core/src/com/fourinachamber/fortyfive/game/enemy/EnemyAction.kt

```diff
@@ -117,10 +119,18 @@ sealed class EnemyActionPrototype(
 
         override fun create(controller: GameController, scale: Double): EnemyAction {
             val amount = (1..maxTurnAmount).random()
-            val rotation = if (Random.nextBoolean()) {
-                GameController.RevolverRotation.Right(amount)
+            val rotation = if (forceDirection == null) {
+                if (Random.nextBoolean()) {
+                    RevolverRotation.Right(amount)
+                } else {
+                    RevolverRotation.Left(amount)
+                }
             } else {
-                GameController.RevolverRotation.Left(amount)
+                when (forceDirection) {
+                    is RevolverRotation.Right -> RevolverRotation.Right(amount)
+                    is RevolverRotation.Left -> RevolverRotation.Left(amount)
+                    else -> RevolverRotation.Right(amount)
```

Comment:
why not just do an "if (forceDirection == RevolverRotation.Left)" ? (like you have with the null check